internal/tests/run/scenario_test.go

```diff
@@ -102,44 +107,21 @@ func TestScenarios(t *testing.T) {
 		err := os.Setenv(randomEnvKey, strconv.FormatUint(rnd, 10))
 		require.NoError(t, err)
 
-		shouldStop := false
-		res, err := run(dirPath)
-		if err != nil {
-			shouldStop = true
-		}
-		t.Log(endInfo)
-
-		if bytes.Contains(res, []byte("FAIL:")) {
-			// buffer
-			buf := bytes.NewBuffer(nil)
-			_, err2 := buf.WriteString(startInfo)
-			require.NoError(t, err2)
-			_, err2 = buf.Write(res)
-			require.NoError(t, err2)
-			_, err2 = buf.WriteString(endInfo)
-			require.NoError(t, err2)
-
-			// store to file
-			errname := fmt.Sprintf("%d_%d.log", rnd, time.Now().Unix())
-			errpath := filepath.Join(path, errname)
-			err2 = os.WriteFile(errpath, buf.Bytes(), 0644)
-			require.NoError(t, err2)
+		// create file
+		errname := fmt.Sprintf("%d_%d.log", rnd, time.Now().Unix())
+		errpath := filepath.Join(path, errname)
+		f, err := os.Create(errpath)
+		require.NoError(t, err)
 
+		err = run(dirPath, f)
+		if err != nil {
```

Comment:
Would be nice to get the error logged so we see it in Drone console log. Don't log non-failing random seed runs so that the console only contains errors.